src/include/daos_srv/evtree.h

```diff
@@ -288,6 +288,7 @@ struct evt_entry_in {
 	uint32_t		ei_inob;
 	/** Address of record to insert */
 	bio_addr_t		ei_addr;
+	bool			ei_corrupted;
```

Comment:
Minor: This ei_corrupted looks redundant, seems we can use the 'corrupted' bit in ei_addr directly.

Reply:
Good catch. Will remove.